sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -1237,5 +1427,132 @@ public virtual async Task<Response> UpdateTranscriptionAsync(string locale, Canc
                 throw;
             }
         }
+
+        /// <summary>
+        /// API to change transcription language.
+        /// </summary>
+        /// <param name="options">An optional object containing update transcription options and configurations.</param>
+        /// <param name="cancellationToken">An optional CancellationToken to cancel the request.</param>
+        /// <returns>Returns an HTTP response with a 202 status code for success, or an HTTP failure error code in case of an error.</returns>
+        public virtual async Task<Response> UpdateTranscriptionAsync(UpdateTranscriptionOptions options = default, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallMedia)}.{nameof(UpdateTranscription)}");
+            scope.Start();
+            try
+            {
+                UpdateTranscriptionRequestInternal request = new(options.Locale)
+                {
+                    OperationContext = options.OperationContext,
+                    SpeechRecognitionModelEndpointId = options.SpeechRecognitionModelEndpointId
```

Comment:
from GA3, this seems to require input Locale. 
is this part of breaking change that is expected?

Reply:
We didn't release the transcription in GA3.